packages/app/src/components/Admin/Security/LocalSecuritySettingContents.jsx

```diff
@@ -30,14 +33,26 @@ class LocalSecuritySettingContents extends React.Component {
     }
   }
 
+  async retrieveIsMailerSetup() {
+    const { adminAppContainer } = this.props;
+    try {
+      await adminAppContainer.retrieveAppSettingsData();
+      this.setState({ isMailerSetup: adminAppContainer.state.isMailerSetup });
+    }
+    catch (err) {
+      toastError(err);
+    }
+  }
+
+  componentDidMount() {
+    this.retrieveIsMailerSetup();
+  }
+
```

Comment:
sorry, please get `isMailerSetup` from `AppContainer.config` instead not adminAppContainer.
you can refer to UserInviteModal.jsx

Reply:
done